modules/apps/frontend-js/frontend-js-react-web/test/js/hooks/useInterval.tsx

```diff
@@ -92,7 +105,7 @@ describe('useInterval()', () => {
 
 		expect(fn).toHaveBeenCalledTimes(1);
 
-		cancel();
+		cancel!();
```

Comment:
Waiting for the day this looks normal to me :)

Reply:
![patience-grasshopper](https://user-images.githubusercontent.com/7074/113693983-80e7be80-96cf-11eb-9381-7bfdf2916f5f.gif)